bignum.c

```diff
@@ -237,6 +237,10 @@ sexp sexp_bignum_fxrem (sexp ctx, sexp a, sexp_sint_t b) {
   b0 = (b >= 0) ? b : -b;
   for (i=len-1; i>=0; i--) {
     n = (n << sizeof(sexp_uint_t)*8) + data[i];
+    // ISSUE_296 - Division by zero, assigned line 237 if b >= 0
+    if (b0 == 0) {
+      return sexp_read_error(ctx, "division by zero", SEXP_NULL, NULL);
```

Comment:
The divide by zero case seems to be caught in sexp_bignum_quot_rem() function, so on inspection it appears 'b' could never be zero in this function.  I didn't see any obvious test case which would invoke this condition.  A cut and paste from sexp_bignum_quot_rem() silences the warning, and seems to be the way to throw a user exception in this case:

```c
  b0 = (b >= 0) ? b : -b;
  if (b0 == 0) {
    return sexp_xtype_exception(ctx, NULL, "divide by zero", a);
  }
```